src/main/java/com/example/demo/tournament/CSVcontroller.java

```diff
@@ -0,0 +1,51 @@
+package com.example.demo.tournament;
+
+import com.example.demo.model.Team;
+import com.opencsv.CSVReader;
+import com.opencsv.CSVReaderBuilder;
+import com.opencsv.CSVWriter;
+import com.opencsv.bean.StatefulBeanToCsv;
+import com.opencsv.bean.StatefulBeanToCsvBuilder;
+import com.opencsv.exceptions.CsvDataTypeMismatchException;
+import com.opencsv.exceptions.CsvRequiredFieldEmptyException;
+import org.springframework.stereotype.Controller;
+
+import java.io.*;
+import java.util.List;
+
+@Controller
+public class CSVcontroller {
+
+    public static void Write(List<Team> listTeams) {
+        String path = "src/main/resources/Students.csv";
+
+        try (Writer writer = new FileWriter(path)) {
+            StatefulBeanToCsv sbc = new StatefulBeanToCsvBuilder(writer)
+                    .build();
+            sbc.write(listTeams);
+        } catch (IOException | CsvDataTypeMismatchException | CsvRequiredFieldEmptyException e) {
+            e.printStackTrace();
```

Comment:
A better way of handling exception, would be to create your custom one, wrap original error in your custom one and rethrow it. This gives you an option to have generic component that handles exceptions: there you can catch it, and print a nice message for user instead of red bloody stacktrace